app/admin/full_submissions.rb

```diff
@@ -33,7 +33,9 @@
                      cc_license_selected submitted_to_registrar]
 
   index do
-    id_column
+    column 'Id', sortable: :id do |submission|
+      link_to submission.id, admin_submission_path(submission.id)
+    end
```

Comment:
This is needed because the automagic `id_column` assumes `Submission#to_param` is the ID field, but this PR switch to using the dissertation ID as the canonical param for submissions. There may be a better way to deal with this but I didn't think an ActiveAdmin deep dive was worth the time since our ActiveAdmin files are already a mess. 🤷🏻‍♂️